tools/topology/topology2/include/components/virtual_widget.conf

```diff
@@ -0,0 +1,46 @@
+#
+# SOF virtual widget
+#
+# A generic volume component. All attributes defined herein are namespaced
+# by alsatplg to "pga.attribute_name"
```

Comment:
@RanderWang still says volume here in the comment. Can you please change to virtual_widget?

Reply:
sorry,  I merged the change into another commit. I will fix it.